api/v1/db/oracledb/member-dao.js

```diff
@@ -166,6 +168,37 @@ const deleteMember = async (memberId) => {
   }
 };
 
+const patchMember = async (memberId, attributes) => {
+  const connection = await conn.getConnection();
+  try {
+    const memberNicknameString = `${attributes.memberNickname ? 'MEMBER_NICKNAME = :memberNickname' : ''}`;
+    const memberEmailString = `${attributes.memberEmail ? 'MEMBER_EMAIL = :memberEmail' : ''}`;
+    const memberPassword = `${attributes.memberPassword ? 'MEMBER_PASSWORD = :memberPassword' : ''}`;
+    const memberLevel = `${attributes.memberLevel ? 'MEMBER_LEVEL = :memberLevel' : ''}`;
+    const memberExpOverLevel = `${attributes.memberExpOverLevel || attributes.memberExpOverLevel === 0 ? 'MEMBER_EXP_OVER_LEVEL = :memberExpOverLevel' : ''}`;
+    const joinedString = _.join(_.compact([memberNicknameString,
+      memberEmailString,
+      memberPassword,
+      memberLevel,
+      memberExpOverLevel,
+    ]), ', ');
```

Comment:
Nice work. Since you're chaining two lodash methods together, could you simplify the code by using the [Lodash chain syntax](https://lodash.com/docs/4.17.14#lodash)?